ethereumetl/domain/receipt.py

```diff
@@ -38,4 +38,5 @@ def __init__(self):
         self.l1_gas_used = None
         self.l1_gas_price = None
         self.l1_fee_scalar = None
-        
+        self.blob_gas_price = None
+        self.blob_gas_used = None
```

Comment:
Shouldn't these 2 new fields be in EthBlock instead of EthReceipt? See here https://eips.ethereum.org/EIPS/eip-4844#header-extension

Reply:
I believe for EthBlock, it would be those 2 fields.
- "blobGasUsed"
- "excessBlobGas"
But, modification of block.py is indeed missing.